packages/fxa-settings/scripts/build.js

```diff
@@ -1,6 +1,39 @@
 // This file was created by react-scripts' (create-react-app) eject script.
 
-'use strict';
+// Here we determine which builds to create. Webpack will configure assets
+// to use a specific URL. In our case, for stage and production builds, we
+// want this be a CDN url, and for dev it can just be the default relative
+// path.
+//
+// This following is controlled by setting the BUILD_TARGETS env. By default
+// we build 'dev', but if you wanted to build stage and prod assets then
+// you'd do this: BUILD_TARGTES=stage,prod yarn build
+const buildTargets = (process.env.BUILD_TARGETS || 'dev').split(',');
+const buildDirTarget = (process.env.BUILD_PATH || 'build/dev').replace(
+  'build/',
+  ''
+);
+if (!buildTargets.includes(buildDirTarget)) {
+  console.log(`Skipping ${buildDirTarget} build.`);
+  return;
+}
+switch (buildDirTarget) {
+  case 'prod':
+    process.env.PUBLIC_URL = 'https://accounts-cdn.moz.aws.net';
+    break;
+  case 'stage':
+    process.env.PUBLIC_URL = 'https://accounts-cdn.stage.moz.aws.net';
+    break;
```

Comment:
I know these URLs don't change often but it would be nice if they are configurable and not hard coded in the build script.

I think this will also affect people who self-host.  Is there a combination of config values that will work for them?

Reply:
I've been thinking about this more. I'll include the dev in the docker build. This should allow people to use the docker image we produce without a CDN.